Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM quay.io/giantswarm/alpine:3.12 AS binaries
+FROM quay.io/giantswarm/alpine:3.14 AS binaries
 
-ARG KUBECTL_VERSION=1.18.9
+ARG KUBECTL_VERSION=1.29.2
 
 RUN apk add --no-cache ca-certificates curl \
     && mkdir -p /binaries \
     && curl -SL https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o /binaries/kubectl \
     && chmod +x /binaries/*
 
-FROM quay.io/giantswarm/alpine:3.12
+FROM quay.io/giantswarm/alpine:3.14
 
 COPY --from=binaries /binaries/* /usr/bin/
 COPY ./kubectl-gs /usr/bin/kubectl-gs
-RUN ln -s /usr/bin/kubectl-gs /usr/bin/kgs
```

Comment:
I would say removing this symlink would break backwards compatibility. I would say we leave this in. Changing the `ENTRYPOINT` is ok though

Reply:
Good point, thanks